tests/integration/ha/test_scaling.py

```diff
@@ -33,6 +33,9 @@
 APP_AND_TLS = [APP_NAME, TLS_CERTIFICATES_APP_NAME]
 
 
+pytestmark = pytest.mark.runner(["self-hosted", "linux", "X64", "jammy", "large"])
+
+
```

Comment:
```suggestion
@pytest.mark.runner(["self-hosted", "linux", "X64", "jammy", "large"])
```

This should be repeated for every test in all integration test modules.

Reply:
Indeed, fixed.